### PR TITLE
Stricter Eslint Part 1

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

Since #553 is controversial, I decided to split in 2, This will be the first part. I think we can agree those changes are good enough to be merged.

### CODE REVIEW

File: frontend/src/components/App/Fishbowl/index.tsx [hunk 1]
```diff
@@ -61,7 +60,7 @@ const Fishbowl: FC = () => {
       category: 'FishbowlReactions',
       label: 'Connect'
     });
-  }, []);
+  }, [fid]);
```
Comment (jordisala1991) [inline frontend/src/components/App/Fishbowl/index.tsx:63]:
```suggestion
  }, []);
```

File: frontend/src/pages/user-conduct-violation.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const UserConductViolationPage = () => {
       prejoin: true,
       conferenceStatus: IConferenceStatus?.NOT_STARTED
     });
-  }, []);
+  }, [dispatch]);
```
Comment (jordisala1991) [inline frontend/src/pages/user-conduct-violation.tsx:31]:
```suggestion
  }, []);
```

File: frontend/src/pages/user-not-participating.tsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ const UserNoParticipatingPage = () => {
       prejoin: true,
       conferenceStatus: IConferenceStatus?.NOT_STARTED
     });
-  }, []); // es
+  }, [dispatch]);
```
Comment (jordisala1991) [inline frontend/src/pages/user-not-participating.tsx:39]:
```suggestion
  }, []); // es
```

File: frontend/src/hocs/withIsFishbowlEnded.js [hunk 4]
```diff
@@ -49,7 +49,7 @@ const withIsFishbowlEnded = WrappedComponent => props => {
         console.log('[STOOA] ', error);
         setError(true);
       });
-  }, []);
+  }, [dispatch, fid, lang, router]);
```
Comment (jordisala1991) [inline frontend/src/hocs/withIsFishbowlEnded.js:52]:
```suggestion
  }, []);
```

File: frontend/src/contexts/StooaManager.tsx [hunk 5]
```diff
@@ -263,6 +263,8 @@ const StooaProvider = ({ data, isModerator, children }) => {
     };
   }, [conferenceStatus]); // eslint-disable-line react-hooks/exhaustive-deps
 
+
+
```
Comment (jordisala1991) [inline frontend/src/contexts/StooaManager.tsx:267]:
```suggestion
```

File: frontend/src/contexts/StooaManager.tsx [hunk 6]
```diff
@@ -263,6 +263,8 @@ const StooaProvider = ({ data, isModerator, children }) => {
     };
   }, [conferenceStatus]); // eslint-disable-line react-hooks/exhaustive-deps
 
+
```
Comment (jordisala1991) [inline frontend/src/contexts/StooaManager.tsx:266]:
```suggestion
```

**Comment by aerrasti** [comment]
so nice that removes all unused variables :tada:

**Review by aerrasti — APPROVED** [review]

**Review by eddsaura — APPROVED** [review]

**Pull request merged**